core_api/src/app.py

```diff
@@ -208,3 +114,9 @@ def embed_sentences(sentences: list[str]) -> EmbeddingResponse:
     """
 
     return model_db.embed_sentences(sentences)
+
+
+file_app = FileSubApp(router, storage_handler, log, publisher, s3, env)
```

Comment:
more explicit use of argument names would be nice style wise (just in case we add further things in future)